CMakeLists.txt

```diff
@@ -134,10 +134,11 @@ add_library(gigamonkey STATIC
     src/gigamonkey/wallet.cpp
     src/gigamonkey/stratum/stratum.cpp
     src/gigamonkey/boost/boost.cpp
+	src/gigamonkey/hash.cpp
```

Comment:
I don't think we need this. Hashes are already taken care of in src/bitcoin_sv/hash.cpp 
Adding this line should cause a conflict actually.

Reply:
No, it doesnt, and yes, iot is needed

Reply:
no,. it isnt